src/pages/components/HistoryItem.qml

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+**
+** Copyright (C) 2020 Open Mobile Platform LLC
```

Comment:
Lower case c is preferred. Noticed that other blocks have capital C but let's use lower case for new ones.